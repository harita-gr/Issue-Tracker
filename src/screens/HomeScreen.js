import React from 'react'
import {Link} from 'react-router-dom';
import { Container,Image,Row,Col, Button } from 'react-bootstrap'
import homeImage from '../images/undraw_fixing_bugs_w7gi.png'
import { LinkContainer } from 'react-router-bootstrap';


function HomeScreen() {
    return (
         <Container>
             <Row>
                 <Col sm={6} md={6} lg={8}>
                   <Image src={homeImage} alt="issue tracker image" className="homeImage mx-auto" />                
                 </Col>
                 <Col sm={10} md={6} lg={4} className="features ">
                    <h4> <i class="fas fa-check-circle"></i> Track Issues </h4> 
                    <h4> <i class="fas fa-check-circle"></i> Visual Reports</h4>
                    <h4> <i class="fas fa-check-circle"></i> User friendly</h4>
                    <LinkContainer to="/signin"> 
                        <Button variant='dark' className='mx-3 px-4'>GET STARTED</Button>
                    </LinkContainer>
                 </Col>
             </Row>
         </Container>
    )
}

export default HomeScreen
