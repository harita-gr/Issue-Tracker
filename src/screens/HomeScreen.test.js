import React from 'react';
import {shallow } from 'enzyme';
import HomeScreen from './HomeScreen'

describe('<HomeScreen/>', () => {

    let wrapper;

    beforeEach(() =>{
        wrapper = shallow(<HomeScreen/>); 
     });

     
        it('should render correctly using Full DOM', () => {
            expect(wrapper).toMatchSnapshot();
        });

});