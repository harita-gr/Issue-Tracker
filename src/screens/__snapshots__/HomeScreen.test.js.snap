// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomeScreen/> should render correctly using Full DOM 1`] = `
<Container
  fluid={false}
>
  <Row
    noGutters={false}
  >
    <Col
      lg={8}
      md={6}
      sm={6}
    >
      <Image
        alt="issue tracker image"
        className="homeImage mx-auto"
        fluid={false}
        rounded={false}
        roundedCircle={false}
        src="undraw_fixing_bugs_w7gi.png"
        thumbnail={false}
      />
    </Col>
    <Col
      className="features "
      lg={4}
      md={6}
      sm={10}
    >
      <h4>
         
        <i
          class="fas fa-check-circle"
        />
         Track Issues 
      </h4>
      <h4>
         
        <i
          class="fas fa-check-circle"
        />
         Visual Reports
      </h4>
      <h4>
         
        <i
          class="fas fa-check-circle"
        />
         User friendly
      </h4>
      <withRouter(LinkContainer)
        to="/signin"
      >
        <Button
          active={false}
          className="mx-3 px-4"
          disabled={false}
          variant="dark"
        >
          GET STARTED
        </Button>
      </withRouter(LinkContainer)>
    </Col>
  </Row>
</Container>
`;
